app/assets/javascripts/test/enzyme/snapshot.e2e.js

```diff
@@ -18,7 +18,6 @@ import mockRequire from "mock-require";
 import test from "ava";
 
 // Those wrappers interfere with global.window and global.document otherwise
-mockRequire("libs/keyboardjs_wrapper", {});
```

Comment:
Are you sure that this change is necessary? The e2e tests failed on circleci :thinking:

Reply:
The `keyboardjs_wrapper` no longer exists, the e2e tests failed because the schema.json couldn't be found, which I fixed in the mean time :)